### ISSUE TITLE
Error with .heif image

### ISSUE THREAD

**Issue by Drzaln** [chlee1001/react-native-simple-heic2jpg#1]
Hi, thank you for this awesome package.
I get this error when uploading .heif image in android
```
err ==>  {
  "nativeStackAndroid": [
    {
      "lineNumber": 110,
      "file": "SimpleHeic2jpgModule.java",
      "methodName": "convertImageAtPath",
      "class": "com.simpleheic2jpg.SimpleHeic2jpgModule"
    },
    {
      "lineNumber": -2,
      "file": "Method.java",
      "methodName": "invoke",
      "class": "java.lang.reflect.Method"
    },
    {
      "lineNumber": 372,
      "file": "JavaMethodWrapper.java",
      "methodName": "invoke",
      "class": "com.facebook.react.bridge.JavaMethodWrapper"
    },
    {
      "lineNumber": 188,
      "file": "JavaModuleWrapper.java",
      "methodName": "invoke",
      "class": "com.facebook.react.bridge.JavaModuleWrapper"
    },
    {
      "lineNumber": -2,
      "file": "NativeRunnable.java",
      "methodName": "run",
      "class": "com.facebook.jni.NativeRunnable"
    },
    {
      "lineNumber": 938,
      "file": "Handler.java",
      "methodName": "handleCallback",
      "class": "android.os.Handler"
    },
    {
      "lineNumber": 99,
      "file": "Handler.java",
      "methodName": "dispatchMessage",
      "class": "android.os.Handler"
    },
    {
      "lineNumber": 27,
      "file": "MessageQueueThreadHandler.java",
      "methodName": "dispatchMessage",
      "class": "com.facebook.react.bridge.queue.MessageQueueThreadHandler"
    },
    {
      "lineNumber": 236,
      "file": "Looper.java",
      "methodName": "loop",
      "class": "android.os.Looper"
    },
    {
      "lineNumber": 228,
      "file": "MessageQueueThreadImpl.java",
      "methodName": "run",
      "class": "com.facebook.react.bridge.queue.MessageQueueThreadImpl$4"
    },
    {
      "lineNumber": 923,
      "file": "Thread.java",
      "methodName": "run",
      "class": "java.lang.Thread"
    }
  ],
  "userInfo": null,
  "message": "Failed to convert image. Bitmap is null.",
  "code": "EUNSPECIFIED"
}
```

this is the file input
```
{
  "fileCopyUri": "file:///data/user/0/com.app.dev/cache/1b48f2ff-204c-4f74-a9ae-d72953ca5a4e/20240910_161417.heic.heif",
  "size": 1552569,
  "name": "20240910_161417.heic.heif",
  "type": "image/heif",
  "uri": "content://com.android.providers.downloads.documents/document/msf%3A226714"
}
```

Can you help me about this issue? Thank you

**Comment by Drzaln** [comment]
I already found the bug and send a [PR](https://github.com/chlee1001/react-native-simple-heic2jpg/pull/2) for this issue

**Comment by chlee1001** [comment]
> I already found the bug and send a [PR](https://github.com/chlee1001/react-native-simple-heic2jpg/pull/2) for this issue

Thank you for your understanding. Although I am currently on vacation, I will do my best to test as soon as possible and proceed with the deployment.

**Comment by Drzaln** [comment]
> 
> Thank you for your understanding. Although I am currently on vacation, I will do my best to test as soon as possible and proceed with the deployment.

Thank you for the update! No rush—enjoy your vacation. I appreciate your time and effort

**Comment by chlee1001** [comment]
Hi,

The issue has been addressed and the fix is now included in version 0.1.2. Could you please update to the latest version and check if the problem is resolved on your end?

Let me know if everything works as expected or if any further issues come up. Thanks again for your patience!
